### ISSUE TITLE
Get-FalconQueue Not Working After Update To v2.1.5

### ISSUE THREAD

**Issue by RandomLegend3** [CrowdStrike/psfalcon#139]
Thanks in advance for your help with this.  Here is the transcript log file.   [PowerShell_transcript.DESKTOP-PGQ8H8T.th3DGG1g.20211106122911.txt](https://github.com/CrowdStrike/psfalcon/files/7490904/PowerShell_transcript.DESKTOP-PGQ8H8T.th3DGG1g.20211106122911.txt)

Expected behavior when I run 'Get-FalconQueue' is to get an updated offline queue.

I was using PSFalcon last night and decided to update to v2.1.5 from v2.1.4.  'Get-FalconQueue' stopped working around after I updated PSFalcon.  Now when I run 'Get-FalconQueue', I get an error pointing to `Public\psf-real-time-response.ps1`:
```
Get-FalconSession : Cannot validate argument on parameter 'Filter'. '(deleted_at:null' is not a valid Falcon Query Language statement.
At C:\Users\cboyle\<redacted>\Documents\WindowsPowerShell\Modules\PSFalcon\2.1.5\Public\psf-real-time-response.ps1:36 char:43
+             $Sessions = Get-FalconSession @SessionParam | Select-Obje ...
+                                           ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-FalconSession], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Get-FalconSession
```
I see in the changelog for v2.1.5 that changes were made to 'Public\psf-real-time-response.ps1' and commands may have moved.  I wonder if that is the issue.  I haven't uninstalled v2.1.5 nor made any changes in fear I will break some other functionality without knowing!

Any thoughts on how to remedy this?

Thanks

**Comment by RandomLegend3** [comment]
Does `Filter` on line 31 always look like that?
```
30           $SessionParam = @{
31                Filter   = "(deleted_at:null+commands_queued:1),(created_at:>'last $Days days'+commands_queued:1)"
32                Detailed = $true
33                All      = $true
34                Verbose  = $true
35            }
36            $Sessions = Get-FalconSession @SessionParam | Select-Object id, device_id
```

**Comment by bk-cs** [comment]
Yes, that is the proper filter. This problem is likely related to [Issue 138](https://github.com/CrowdStrike/psfalcon/issues/138); the 'FQL testing' that I implemented into v2.1.5 is too strict. I'll review and update when I have a fix available, but until then, you should be able to modify `Public\real-time-response.ps1` and change the `Get-FalconSession` command so that the `ValidateScript` property is removed from the `Filter` parameter.

Before:
```
function Get-FalconSession {
    [CmdletBinding(DefaultParameterSetName = '/real-time-response/queries/sessions/v1:get')]
    param(
        [Parameter(ParameterSetName = '/real-time-response/entities/queued-sessions/GET/v1:post',
            Mandatory = $true, Position = 1)]
        [Parameter(ParameterSetName = '/real-time-response/entities/sessions/GET/v1:post', Mandatory = $true,
            Position = 1)]
        [ValidatePattern('^\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$')]
        [array] $Ids,

        [Parameter(ParameterSetName = '/real-time-response/queries/sessions/v1:get', Position = 1)]
        [ValidateScript({ Test-FqlStatement $_ })]
        [string] $Filter,
```

After:
```
function Get-FalconSession {
    [CmdletBinding(DefaultParameterSetName = '/real-time-response/queries/sessions/v1:get')]
    param(
        [Parameter(ParameterSetName = '/real-time-response/entities/queued-sessions/GET/v1:post',
            Mandatory = $true, Position = 1)]
        [Parameter(ParameterSetName = '/real-time-response/entities/sessions/GET/v1:post', Mandatory = $true,
            Position = 1)]
        [ValidatePattern('^\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$')]
        [array] $Ids,

        [Parameter(ParameterSetName = '/real-time-response/queries/sessions/v1:get', Position = 1)]
        [string] $Filter,
```

**Comment by RandomLegend3** [comment]
@bk-cs that did the trick!  Thank you :)
